index.html

```diff
@@ -2044,6 +2146,76 @@ <h2 id="links">Links &amp; Suggested Reading</h2>
 
       <h2 id="changelog">Change Log</h2>
 
+      <p id="1.8.0">
+        <b class="header">1.8.0</b> &mdash; <small><i>Jan 23, 2015</i></small> &mdash; <a href="https://github.com/jashkenas/underscore/compare/1.7.0...1.8.0">Diff</a> &mdash; <a href="https://cdn.rawgit.com/jashkenas/underscore/1.8.0/index.html">Docs</a><br />
+        <ul>
+          <li>
+            Added <tt>_.keysIn</tt> which returns the enumerable properties of an object.
+          </li>
+          <li>
+            Reverted change where <tt>_.extend</tt> only iterates own properties, it now uses <tt>_.keysIn</tt>. Added <tt>_.assign</tt> that aligns closely with ECMA6 <tt>Object.assign</tt> and iterates over own properties.
+          </li>
+
+          <li>
+            Added <tt>_.create</tt> helper for simpler inheritance patterns.
+          </li>
+
+          <li>
+            Added extremely handy <tt>_.transform</tt> method as an alternative to <tt>_.reduce</tt> in many cases.
```

Comment:
"extremely handy" is a bit much here. Doesn't really fit with the rest of the changelog